translations/trading.tex/pl.po

```diff
@@ -37,31 +37,36 @@ msgstr ""
 
 #.  !TeX spellcheck = en_US
 #. type: Plain text
-#: sections/other_rules.tex:4
-msgid "\\addsection{Other Rules}{\\skills/armorer.png}"
-msgstr "\\addsection{Inne Zasady}{\\skills/armorer.png}"
+#: sections/trading.tex:4
+#, fuzzy
+#| msgid "\\addsection{Town}{\\skills/artillery.png}"
+msgid "\\addsection{Trading}{\\skills/armorer.png}"
+msgstr "\\addsection{Miasto}{\\skills/artillery.png}"
```

Comment:
```suggestion
msgid "\\addsection{Trading}{\\skills/armorer.png}"
msgstr "\\addsection{Handel}{\\skills/armorer.png}"
```
No, it doesn't. I don't know why `po4a` does this weird stuff with fuzzy translations.